artiq/coredevice/i2c.py

```diff
@@ -137,8 +142,10 @@ def i2c_read_many(busno, busaddr, addr, data):
         i2c_stop(busno)
 
 
-class PCA9548:
-    """Driver for the PCA9548 I2C bus switch.
+class PCA954X:
```

Comment:
Just call it I2CSwitch (and same for the syscall). We might use others in the future.